src/Application/Ucan.elm

```diff
@@ -25,7 +25,7 @@ fsResourceKey =
 
 {-| <https://whitepaper.fission.codes/access-control/ucan/webnative-attenuation#capabilities>
 -}
```

Comment:
Just to make sure that we're on the same page for the closed UCAN on 0.3.1 — we don't have an array — it's only the one resource (WNFS). The workaround is that there is an existing `AuthNOnly` capability (in JSON that's a `null`).

Reply:
Hmm, I don't follow. I don't think I'm assuming an array 🤔 Did you change the resource key from `floofs` to `wnfs`?

Reply:
> Hmm, I don't follow. I don't think I'm assuming an array 🤔 Did you change the resource key from floofs to wnfs?

I don't think so, but lemme check the Aeson instances.

I'm more saying that the only way to do a closed UCAN in 0.3.x is with `ptc: null`, since we can't `att: []`. I just wanted to make sure that was the case, but I can trawl through the code now 👍

Reply:
Oh ok, gotcha. `ptc` is set to `undefined` at the moment, so there's no `ptc` at all in the payload. I can set to `null` so the key is present in the object if you need that?

Reply:
> lemme check the Aeson instances

Indeed still `floofs` on 0.3

![8DF53347-63C2-4988-94B9-F68B8D97BE53](https://user-images.githubusercontent.com/1052016/97926566-3d4fe980-1d18-11eb-91a3-90433348c022.jpeg)